scripts/components/Login.js

```diff
@@ -0,0 +1,61 @@
+import btoa from "btoa";
+import localStorage from "localStorage";
+import React from "react";
+import { Link } from "react-router";
+
+export class LoginForm extends React.Component {
```

Comment:
I like the idea of handling logins, but it doens't match with what @QuentinRoy and I had in mind.

We should explicit the login flow, but the main idea would be to:
- Create a project with a mnemonic name (easy to remember). Also possible to have the user define it if they want;
- Make it possible to read the project information from the has part of the URI.